libfreerdp/core/message.c

```diff
@@ -244,26 +244,11 @@ static BOOL update_message_SurfaceBits(rdpContext* context,
 	if (!context || !context->update || !surfaceBitsCommand)
 		return FALSE;
 
-	wParam = (SURFACE_BITS_COMMAND*) malloc(sizeof(SURFACE_BITS_COMMAND));
+	wParam = copy_surface_bits_command(context, surfaceBitsCommand);
 
 	if (!wParam)
 		return FALSE;
 
-	CopyMemory(wParam, surfaceBitsCommand, sizeof(SURFACE_BITS_COMMAND));
-#ifdef WITH_STREAM_POOL
```

Comment:
Not quite sure but there is one more place in code that uses #ifdef WITH_STREAM_POOL.
As WITH_STREAM_POOL is defined on top of the source file I think this line should be removed as well as this could lead to some unwanted behavior?